libraries/abstractions/common_io/test/test_iot_spi.c

```diff
@@ -681,7 +755,7 @@ TEST( TEST_IOT_SPI, AFQP_IotSPI_ReadSyncFuzzing )
     TEST_ASSERT_NOT_EQUAL( NULL, xSPIHandle );
 
     lRetVal = iot_spi_read_sync(NULL, ucBuffer, SPI_BUFFER_SIZE);
-    TEST_ASSERT_EQUAL( IOT_SPI_INVALID_VALUE, lRetVal );
```

Comment:
Following the same behavior of all the other api's, if we pass in a NULL handle the return is INVALID_VALUE, not NOT_INITIALIZED.

Reply:
I agree with this, what is the purpose for IOT_SPI_MASTER_NOT_INITIALIZED in the API? Should we just remove it if we are not going to use it?